modules/tinymce/src/themes/silver/main/ts/ui/general/Tooltips.ts

```diff
@@ -1,16 +1,19 @@
-import { GuiFactory, Tooltipping } from '@ephox/alloy';
-
-import { UiFactoryBackstageShared } from '../../backstage/Backstage';
+import { AlloyComponent, GuiFactory, Tooltipping } from '@ephox/alloy';
+import { Fun, Result } from '@ephox/katamari';
 
 type TooltippingBehaviour = ReturnType<typeof Tooltipping['config']>;
 
 // TODO: Consider using this once we fix the delayed attempt to appear after it's gone problem.
 
+// FIX this with a proper design / architecture.
+let numActiveTooltips = 0;
+
 // TODO: Make the arrow configurable.
-const upConfig = (item: { text: string }, sharedBackstage: UiFactoryBackstageShared): TooltippingBehaviour => Tooltipping.config({
-  delay: 200,
+const upConfig = (item: { text: string }, getSink: () => Result<AlloyComponent, any>): TooltippingBehaviour => Tooltipping.config({
+  delayForShow: () => numActiveTooltips > 0 ? 0 : 800,
```

Comment:
Can we have a common default value for `delayForShow` and `delayForHide` (800)? Instead of declaring it here and in `ToolTipBackStage.ts`

Reply:
This file isn't used. It's going to be deleted.